app/controllers/admin/demo_nights_controller.rb

```diff
@@ -1,4 +1,11 @@
 class Admin::DemoNightsController < Admin::BaseController
+  before_action :check_active_demo_nights, only: :create
+
+  def index
+    demo_nights = DemoNight.all
+    @inactives = demo_nights.where(status: "closed")
+    @currents = demo_nights.where.not(status: "closed")
```

Comment:
these would be instance methods, so we'd have to iterate through, we could pass the whole array of `@demo_nights` to the method to avoid the call?